RichardSkillsLibrary.py

```diff
@@ -13,8 +13,9 @@
 # Define Class Robot
 class Richard_Skills:
 
-    #color_sensor_left = ColorSensor(Port.S3)
-    #color_sensor_right = ColorSensor(Port.S4)
+    ultrasonic_sensor = UltrasonicSensor(Port.S1)
```

Comment:
I think this will cause the `main.py` to break for people who don't have an Ultrasonic sensor in Port 1. Could you instead put this inside one of the functions, so it only runs this if you call the function?

For example:

```python
def detect_wall(self, distance_mm, angle_turn):
    ultrasonic_sensor = UltrasonicSensor(Port.S1)
    # regular stuff
```

Reply:
I updated it.